src/api/app.js

```diff
@@ -12,6 +24,17 @@ export default {
     }
     xhr.send()
   },
+  getGithubUsername: function (email, callback) {
+    const url = `https://api.github.com/search/commits?q=author-email:${email}&sort=author-date&per_page=1`
```

Comment:
Does the `api` have some daily limits?

Reply:
Yes, Github has a rate limit of 5000 requests per hour as seen [here ](https://developer.github.com/apps/building-integrations/setting-up-and-registering-github-apps/about-rate-limits-for-github-apps/).

If this site has that much traffic then we can improve it a bit by adding token else no cause of alarm.